cmd/installer/installer.go

```diff
@@ -205,37 +244,94 @@ func installBashCompletionScript(cifuzzPath string) error {
 			return errors.WithStack(err)
 		}
 		cmd = exec.Command("bash", "-c", "'"+cifuzzPath+"' completion bash > \""+dir+"/cifuzz\"")
+		log.Printf("Command: %s", cmd.String())
+		err = cmd.Run()
+		if err != nil {
+			return errors.WithStack(err)
+		}
 	case "darwin":
-		cmd = exec.Command("bash", "-c", "'"+cifuzzPath+"' completion bash > \"$(brew --prefix)/etc/bash_completion.d/cifuzz\"")
+		// There are no bash completoin directories on macOS by default,
+		// so we need user action to source our installation directory
+		notes = append(notes, fmt.Sprintf(`To enable command completion:
+
+    # enable bash completion (if not already enabled):
+    echo "if [ -f $(brew --prefix)/etc/bash_completion ]; then . $(brew --prefix)/etc/bash_completion; fi" >> ~/.bash_profile
```

Comment:
Suggestion for minor improvements:
* Use the more descriptive `source` instead of `.`
* Use the shorter `&&` instead of `if`, `then` (since this is a one-liner - otherwise I also prefer `if`)
```suggestion
    echo "[ -f $(brew --prefix)/etc/bash_completion ] && source $(brew --prefix)/etc/bash_completion" >> ~/.bash_profile
```

Reply:
Great suggestion, will change. Thanks!